recipe/meta.yaml

```
@@ -44,18 +44,23 @@ requirements:
     - importlib-metadata
     - importlib-resources
     - typing-extensions
-    - numpy
+    - numpy >=1.21.0
 
 test:
+  source_files:
+    - tests/
   imports:
     - ale_py
   requires:
     - pip
+    - pytest
+    - gymnasium >=1.0.0
```

is this an optional dependency? Then we can do 
```suggestion
    - gymnasium >=1.0.0  # [py<313]
```
until gymnasium has 3.13 support. If it's non-optional, it needs to move to the run requirements.